jobs/credhub/spec

```diff
@@ -277,6 +277,12 @@ properties:
   credhub.certificates.concatenate_cas:
     description: "Enables the concatenation of CAs when there is a transitional CA for a certificate."
     default: true
+  credhub.certificates.ca_minimum_duration_in_days:
+    description: "CA certificates will be generated/regenerated with this value when the user provided duration is lower. Unit in days."
```

Comment:
Slight language fix to make it clearer.

The feature is somewhat convoluted, the documentation should be very clear about what this does.